source/connect/tls.txt

```diff
@@ -0,0 +1,390 @@
+.. _pymongo-tls:
+
+=============
+Configure TLS 
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: security, authentication, transport layer security, encrypt
+
+Overview
+--------
+
+In this guide, you can learn how to use the :wikipedia:`TLS <Transport_Layer_Security>`
+protocol to secure your connection to a MongoDB deployment.
+
+When you enable TLS for a connection, {+driver-short+} uses TLS to connect to the
+MongoDB deployment, verifies the deployment's certificate, and ensures that the certificate
+certifies the deployment {+driver-short+} is connecting to.
```

Comment:
Not dead-set on this suggestion, but I think it could be more clear in list form:

```suggestion
When you enable TLS for a connection {+driver-short+} performs the following actions:

- Uses TLS to connect to the MongoDB deployment
- Verifies the deployment's certificate
- Ensures that the certificate certifies the deployment {+driver-short+} is connecting to
```